### PR TITLE
Remove rating/score columns for interactive scenes (MCS-621).

### PR DESCRIPTION
**Pull request by rartiss55**
Analysis UI page update: removing rating and score columns from score table for interactive scenes since they don't apply there.

### CODE REVIEW

File: analysis-ui/src/components/Analysis/scenes.jsx [hunk 1]
```diff
@@ -74,7 +74,14 @@ const scoreTableCols = [
     { dataKey: 'score.confidence', title: 'Score/Confidence', dataType: 'history'}
 ]
 
-const scoreTableColsWithCorners = scoreTableCols.concat([{ dataKey: 'corner_visit_order', title: 'Corner Visit Order', dataType: 'history'}])
+const scoreTableColsInteractive = [
```
Comment (rartiss55) [inline analysis-ui/src/components/Analysis/scenes.jsx:77]: I think hardcoding the values in the score table component instead might be preferable, but wanted to hold off on refactoring that until slices were available on the history records so I can clean some of that logic up too (we can get rid of the dataType key here in that case).

Comment (brianpippin) [reply analysis-ui/src/components/Analysis/scenes.jsx:77]: I just merged the code for the script that addes slices, if you want to run that locally so you can finish up the part you mentioned.

**Review by brianpippin — APPROVED** [review]
Code looks good.  Will rereview if you add the new slice stuff or create a new ticket for that if you want.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Review by jaudick — APPROVED** [review]

**Comment by rartiss55** [comment]
> Code looks good. Will rereview if you add the new slice stuff or create a new ticket for that if you want.

Thanks @brianpippin -- added the slice stuff in and nixed some of the refactoring I had in mind since I forgot about the eval 2 case. Also added minor fixes to get the eval 2 analysis page working again.

File: analysis-ui/src/components/TestOverview/evalNavItem.jsx [hunk 2]
```diff
@@ -61,7 +61,8 @@ class EvalNavItem extends React.Component {
 
                     // Remove Evaluation 2 Results because this page is designed to show hypercube results
                     //    and we did not have a hypercube design for Evaluation 2
-                    evalOptions = evalOptions.filter(element => element.label !== "Evaluation 2 Results");
+
+                    evalOptions = evalOptions.filter(element => element.label !== "eval_2_results");
```
Comment (brianpippin) [inline analysis-ui/src/components/TestOverview/evalNavItem.jsx:65]: I have to pull this out and test, but I think the label will still be "Evaluation 2 Results" and the value will be "eval_2_results" we should test this one.

Comment (rartiss55) [reply analysis-ui/src/components/TestOverview/evalNavItem.jsx]: Ah, you're right! Fixed now.

Comment (rartiss55) [reply analysis-ui/src/components/TestOverview/evalNavItem.jsx]: Ah, you're right! Fixed now.

**Review by brianpippin — APPROVED** [review]

**Pull request merged**